### PR TITLE
feat: add endpoints command

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

## Proposed changes
- Adds endpoints command
-

### CODE REVIEW

File: pkg/model/manifest.go [hunk 1]
```diff
@@ -474,6 +474,7 @@ func inferHelmTags(path string) string {
 		}
 		if image.Image != "" {
 			result += fmt.Sprintf(" --set %s.image=${OKTETO_BUILD_%s_IMAGE}", svcName, strings.ToUpper(svcName))
+			os.Setenv(fmt.Sprintf("OKTETO_BUILD_%s_IMAGE", strings.ToUpper(svcName)), image.Image)
```
Comment (pchico83) [inline pkg/model/manifest.go:477]: is this change related to this PR?

**Review by pchico83 — APPROVED** [review]

File: pkg/model/manifest.go (hunk 1 above)
Comment (jLopezbarb) [reply pkg/model/manifest.go:477]: not related, but it's a bug that needs to be fixed. When we deploy a helm chart without Okteto Manifest we were not setting that env var which was causing an error
